config/locales/crowdin/fi.yml

```diff
@@ -153,9 +153,9 @@ fi:
       add: 'Lisää toiminto'
       assigned_to:
         executing_user_value: '(Assign to executing user)'
-    conditions: 'Olosuhteet'
-    plural: 'Custom actions'
-    new: 'New custom action'
+    conditions: 'Ehdot'
```

Comment:
Why do you have changes in the finnish translations? This should not be part of this PR.